docs/administration.md

```diff
@@ -45,34 +45,283 @@ to put the account into the "inactive" state. Deleting accounts is not
+```
+The response will look something like:
+```json
+{
+  "active": [
+    {
+      "address": "9wRAu3giCtKhSsVnkZJ7LLE6zqzrmMKpPg39S8aoC7T6F6GobeDpz8TcvVfTQT3ucW82oTYKG8v3ZMAeh8SZVXWwMdvwZew",
+      "scan_height": 2220875,
+      "access_time": 1681244149
+    }
+  ]
+}
+```
+
+### list_requests
+This is a listing of all pending new account requests and all requests
+to import from genesis block requests. When auth is disabled usage
+looks like:
+```bash
+curl -v -H "Content-Type: application/json" -d '{}' http://127.0.0.1:8081/list_requests
+```
+and with auth enabled looks like:
+```bash
+curl -v -H "Content-Type: application/json" -d '{"auth": "f50922f5fcd186eaa4bd7070b8072b66fea4fd736f06bd82df702e2314187d09"}' http://127.0.0.1:8081/list_requests
+```
+### modify_account_status
+This can change an account status to `active`, `inactive` or `hidden`. The
+`active` state is the normal state - the account is being scanned and
+returned by the API. The `inactive` state is still returned by the API,
+but is no longer being scanned. The `hidden` is the current way to
+"delete" an account - it is not scanned nor returned by the API. Accounts
+cannot currently be deleted due to internal DB requirements.
+
+### reject_requests
+This is the opposite of [`accept_requests`](#accept_requests) above. See
+information from that endpoint on how to use this one.
+
+### rescan
+This tells the scanner to rescan specific account(s) from the specified
+height.
+
+### webhook_add
+This is used to track a specific payment ID to an address or all general
+payments to an address (where payment ID is zero). Using this endpint requires
+a web address for callback purposes, a primary (not integrated!) address, and
+finally the type ("tx-confirmation"). The event will remain in the database
+until one of the delete commands ([webhook_delete_uuid](#webhook_delete_uuid)
+or [webhook_delete](#webhook_delete)) is used to remove it.
+
+> The provided URL will use SSL/TLS if `https://` is prefixed in the URL and
+will use plaintext if `http://` is prefixed in the URL. SSL/TLS connections
+will use the system certificate authority (root-CAs) by default, and will
+ignore all authority checks if `--webhook-ssl-verification none` is provided
+on the command line when starting `monero-lws-daemon`. The webhook will fail
+if there is a mismatch of `http` and `https` between the two servers, and
+will also fail if `https` verification is mismatched. The rule is: (1) if
+the callback server has SSL/TLS disabled, the webhook should use `http://`,
+(2) if the callback server has a self-signed certificate, `https://` and
+`--webhook-ssl-verification none` should be used, and (3) if the callback
+server is using "Let's Encrypt" (or similar), then `https://` with no
+additional command line flag should be used.
+
+
+#### Initial Request to server
+Example where admin authentication is required (`--disable-admin-auth` NOT set on start):
```

Comment:
Could you rephrase that to include the default behavior of monero-lws? :)

Reply:
Done.